### PR TITLE
Testinggems

### PR DESCRIPTION
**Pull request by Tonyynot14**
Added in gems related to rspec for testing in the future. This is a commit so if anyone wants to run test they will be able to from now on, Users will need to install phantom.js into the path still

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -40,6 +40,8 @@ gem 'bootsnap', '~>1.4.1'
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug'
+  #RSPEC gem for testing 
+  gem 'rspec-rails'
```
Comment (leila-alderman) [inline Gemfile]: Please specify versions for all of the gems that you add.

File: Gemfile [hunk 2]
```diff
@@ -50,3 +52,10 @@ group :development do
   gem 'spring'
 end
 
+group :test do
+  # Adds support for Capybara system testing and selenium driver
+  gem 'capybara', '>= 2.15'
+  gem 'poltergeist'
```
Comment (leila-alderman) [inline Gemfile]: Could you explain why you're requiring poltergeist here? It seems to be an older, no longer supported gem that isn't needed for capybara to work.

**Review by leila-alderman — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 2 above)
Comment (Tonyynot14) [reply Gemfile]: From the class we took with Ramiro poltergeist is used to communicate to make phantonjs headless browser function.

Comment (Tonyynot14) [reply Gemfile]: https://github.com/teamcapybara/capybara#poltergeist
Is a link to it, It just sets up a headless browser with phantomjs headless browser.

**Comment by Tonyynot14** [comment]
Going to stick with phantomjs for now because I know how it works. I will look more into capybara-webkit for future testing

**Review by leila-alderman — APPROVED** [review]

**Pull request merged**